crux/src/Crux/Log.hs

```diff
@@ -177,8 +185,8 @@ cruxLogMessageToSayWhat SimulationTimedOut =
 cruxLogMessageToSayWhat SkippingUnsatCoresBecauseMCSatEnabled =
   cruxWarn "Warning: skipping unsat cores because MC-SAT is enabled."
 
-cruxLogMessageToSayWhat (StartedGoal goalNumber) =
-  cruxOK (T.pack ("Started goal " ++ show goalNumber))
```

Comment:
This message already existed; is this an intentional behavioral change for existing uses?

Reply:
I accidentally introduced this message in 970ed02d76ca75e95ba8bdfee490fffeebba917e.

I don't think it corresponded to anything pre-existing, and so this message was actually never printed except in my personal logging extension attempts. I had meant to stash this away, but missed it when shuffling things around.